canonical-web-and-design/anbox-cloud.io
# Build anbox-cloud landing page

## Done

- Build new Anbox cloud landing page
- Some links are missing as these pages are yet to be built: `Learn more about virtual devices`, `Learn more about secure apps`, `Learn more about game streaming`, `Learn more about automated testing`
- Blocked: Missing link `Deploy on AWS` - waiting for response from Galem.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/1SfpOBeDJUZj2ZYq4V8lHOUbQF9Yzo7mp5NajJDVwFMI/edit#heading=h.pnlo30ndk43n) 


## Issue / Card

Fixes [#4006](https://github.com/canonical-web-and-design/web-squad/issues/4006)

## Screenshots
